jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -69,16 +70,59 @@ def _to_np_dtype(dtype):
       tf_unimpl(f"{prim.name} is unimplemented for dtype {np_dtype}")
 
   if prim in [lax.rem_p, lax.atan2_p]:
-    # b/158006398: TF kernels are missing for 'rem' and 'atan2'
     np_dtype = _to_np_dtype(args[0].dtype)
     if np_dtype in [np.float16, dtypes.bfloat16]:
+      # b/158006398: TF kernels are missing for 'rem' and 'atan2'
       tf_unimpl(f"Missing TF kernels for {prim.name} with dtype {np_dtype}")
```

Comment:
I think we should change the text to "{prim.name} is unimplemented ...", like the others

Reply:
Done in `99a745e`